tools/integration-test/src/tests/connection_delay.rs

```diff
@@ -0,0 +1,106 @@
+use core::time::Duration;
+use time::OffsetDateTime;
+
+use ibc_test_framework::ibc::denom::derive_ibc_denom;
+use ibc_test_framework::prelude::*;
+use ibc_test_framework::util::random::random_u64_range;
+
+const CONNECTION_DELAY: Duration = Duration::from_secs(10);
+
+#[test]
+fn test_connection_delay() -> Result<(), Error> {
+    run_binary_channel_test(&ConnectionDelayTest)
+}
+
+pub struct ConnectionDelayTest;
+
+impl TestOverrides for ConnectionDelayTest {
```

Comment:
Seems like that `max_delay_block` in the integration test defaults to `0` instead of `10s`.